src/resources/i18n/translations/en.json

```diff
@@ -463,51 +462,51 @@
 	"Podverse External Storage Permission": "Podverse permission to external storage card",
 	"Podverse would like to access your device's external storage to access your downloaded media.": "Grant permission to access your downloaded media.",
 	"Podverse would like to access your device's external storage to store downloaded media.": "Grant permission to store downloaded media.",
-	"Popularity Analytics": "Popularity tracking",
-	"Premium Info": "Podverse premium info",
-	"Premium Membership Expired": "Renew Podverse premium first",
-	"Premium Membership Required": "Podverse premium required",
-	"Premium": "Podverse premium",
+	"Popularity Analytics": "Popularity Tracking",
+	"Premium Info": "Podverse Premium Info",
+	"Premium Membership Expired": "Renew Podverse Premium first",
+	"Premium Membership Required": "Podverse Premium required",
+	"Premium": "Premium",
```

Comment:
```suggestion
	"Premium": "Podverse Premium",
```

Reply:
In this case "Premium" is the column header in a comparison table. We'd prefer the shorter "Premium" here.

![image](https://user-images.githubusercontent.com/2262919/153319630-742e3a8a-95cc-4051-bcfb-32dfdf4c2c59.png)